.gitignore

```diff
@@ -0,0 +1 @@
+**/__pycache__
```

Comment:
Please configure your IDE to put a line at the end of each file.

Reply:
Done